### ISSUE TITLE
NPM package v3.3.7 doesn't contain solar files

### ISSUE THREAD

**Issue by chriswhite199** [thomaspark/bootswatch#643]
The NPM package for v3.3.7 doesn't contain the solar files:

    curl -s https://registry.npmjs.org/bootswatch/-/bootswatch-3.3.7.tgz | tar tvz | grep -E '(cyborg|solar)'
    -rw-r--r--  0 501    20       3924 Jul 30  2016 package/cyborg/_bootswatch.scss
    -rw-r--r--  0 501    20      30963 Jul 30  2016 package/cyborg/_variables.scss
    -rw-r--r--  0 501    20     149812 Jul 30  2016 package/cyborg/bootstrap.css
    -rw-r--r--  0 501    20     125625 Jul 30  2016 package/cyborg/bootstrap.min.css
    -rw-r--r--  0 501    20       3961 Jul 30  2016 package/cyborg/bootswatch.less
    -rw-r--r--  0 501    20      25410 Dec  6  2014 package/cyborg/thumbnail.png
    -rw-r--r--  0 501    20      27361 Jul 30  2016 package/cyborg/variables.less

**Comment by lord-alfred** [comment]
Confirm also met a error when compiling SASS this theme.

**Comment by Drapegnik** [comment]
+1

**Comment by svenevs** [comment]
So fixing this ex-post-facto isn't really simple assuming npm (don't use it, don't know) is using the actual tags on GitHub, [tag `v3.3.7` is the problem](https://github.com/thomaspark/bootswatch/tree/v3.3.7).

I'm not sure but I think this could be fixed with a `v3.3.7+1` tag?  Previous tags had similar things:

```
v3.3.5
v3.3.5+1
v3.3.5+2
v3.3.5+3
v3.3.5+4
v3.3.6
v3.3.6+1
v3.3.6+2
```

Not sure how things are getting distributed where, but solar is way too good to not be in there :blush:

**Comment by thomaspark** [comment]
Unfortunately npm will strip out the build number (aka everything after the `+`), unless something changed recently.
